includes/class-wc-payments.php

```diff
@@ -1319,8 +1319,11 @@ function wcpay_show_old_woocommerce_for_norway_notice() {
 					<p>
 					<?php
 					echo WC_Payments_Utils::esc_interpolated_html(
-						/* translators: %s: documentation URL */
-						__( 'The WooCommerce version you have installed is not compatible with WooCommerce Payments for a Norwegian business. Please update WooCommerce to version 7.5 or above. You can do that via the <a1>the plugins page.</a1>', 'woocommerce-payments' ),
+						sprintf(
+							/* translators: %s: WooPayments, a1: documentation URL */
+							__( 'The WooCommerce version you have installed is not compatible with %s for a Norwegian business. Please update WooCommerce to version 7.5 or above. You can do that via the <a1>the plugins page.</a1>', 'woocommerce-payments' ),
```

Comment:
Nitpick: We could also externalise `WooCommerce` to a `%s` value for the translation here. 🙂

Reply:
Thanks! Fixed in https://github.com/Automattic/woocommerce-payments/pull/6581/commits/a10101f8f95ed321704be33f29bad8ac81c0aa01